DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: hazardItems
 Type: Package
 Title: hazardItems
-Version: 1.3.7
+Version: 1.3.8
```

Comment:
I think we'll need to get to 1.3.9 for the release, this is causing merge conflict